pkg/pillar/hypervisor/kvm.go

```diff
@@ -268,6 +268,13 @@ const qemuDiskTemplate = `
   addr = "0x0"
 {{- end}}
   drive = "drive-virtio-disk{{.DiskID}}"
+{{- else}}
+[device "vhost-disk{{.DiskID}}"]
+  driver = "vhost-scsi-pci"
+  wwpn = "{{.WWN}}"
+  bus = "pci.{{.PCIId}}"
+  addr = "0x0"
+{{- end}}
```

Comment:
You should add a test for this to kvm_test.go

Reply:
Added